README.md

```diff
@@ -1 +1,254 @@
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Perform Calculation]**
+- [ ] **[Generate Quotes]**
+- [ ] **[Project Ui Developement]**
+- [ ] **[Ui Developement Improvements]**
+- [ ] **[Application deployment]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/RileyManda/math-magicians/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+<!-- SUPPORT -->
+## <b>Show your support 🌟</b><a name="support"></a>
+
+Thank you for taking the time to explore this project! Your support means a lot to me. If you find my project valuable and would like to contribute, here is one way you can support me:
+
+ - <b>Star the project ⭐️</b>: Show your appreciation by starring this GitHub repository. It helps increase visibility and lets others know that the project is well-received.
+
+ - <b>Fork the project 🍴 🎣</b>: If you're interested in making improvements or adding new features, feel free to fork the project. You can work on your own version and even submit pull requests to suggest changes.
+
+ - <b>Share with others 🗺️</b>: Spread the word about this project. Share it on social media, mention it in relevant forums or communities, or recommend it to colleagues and friends who might find it useful.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my sincere gratitude to [Microverse](https://github.com/microverseinc), the dedicated reviewers, and collaborators. Your unwavering support, feedback, and collaborative efforts have played an immense role in making this journey a resounding success. I am truly grateful for your contributions and for being an integral part of my achievements. Thank you for your continued support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Question_1**
+
+  Do I have to use the vs code specifically?
+
+  - Answer_1
+
+    You can use any code editor of your choice. <br>
+
+- **Question_2**
+
+  Where can I download node JS for installation?
+
+  - Answer_2
+
+  Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.
+  It can be downloaded [here](https://nodejs.org/en/download/Node.jsNode.js): 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ATTRIBUTION -->
+
+## 👥 Attribution <a name="attribution"></a>
+- ProjectIcon: [Svgrepo](https://www.svgrepo.com/)
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](./LICENSE)
```

Comment:
- [ ] Kindly fix the license link in  your readme, it currently shows a  404 error when I click the link 👍🏿

Reply:
Thank you @prolajumokeoni .
I will make sure to fix this. I see I forgot to include the License in my project and this is resulting in a broken path.